src/Repository/JobsRepository.php

```diff
@@ -21,22 +21,89 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Jobs::class);
     }
 
-    public function save(Jobs $entity, bool $flush = false): void
+    public function save(Jobs $entity): bool
     {
         $this->getEntityManager()->persist($entity);
+        $this->getEntityManager()->flush();
 
-        if ($flush) {
-            $this->getEntityManager()->flush();
-        }
+        return $entity->getId() > 0;
     }
 
-    public function remove(Jobs $entity, bool $flush = false): void
+    public function remove(Jobs $entity): void
     {
         $this->getEntityManager()->remove($entity);
+        $this->getEntityManager()->flush();
+    }
+
+    public function update(int $id, array $params): int
+    {
+        $queryBuilder = $this->createQueryBuilder('j');
+
+        $nbUpdatedRows = $queryBuilder->update()
+            ->set('j.name', ':jobName')
+            ->set('j.description', ':jobDescription')
+            ->where('j.id = :jobId')
+            ->setParameter('jobName', $params['name'])
+            ->setParameter('jobDescription', $params['description'])
+            ->setParameter('jobId', $id)
+            ->getQuery()
+            ->execute();
+
+        return $nbUpdatedRows;
+    }
+
+    public function select(): array
+    {
+        $queryBuilder = $this->createQueryBuilder('j');
+
+        $selectedRows = $queryBuilder
+            ->getQuery()
+            ->getArrayResult();
```

Comment:
change ->getArrayResult() to ->getResult(). This way the result array of the query will contain the company related information.